test/feedback.spec.js

```diff
@@ -0,0 +1,86 @@
+import chai, { expect, request, use } from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../src/app';
+import models from '../src/database/models';
+
+const { Feedback, Users } = models;
+chai.should();
+use(chaiHttp);
+
+let loginToken;
+const feedback = {
+  feedback: 'feedback'
+};
+
+before(async () => {
+  const res = await request(server).post('/api/user/login').send({
+    email: 'demouser2@cod.be',
+    password: 'altp6@random'
+  });
+  loginToken = res.body.accessToken;
+});
+
+after(async () => {
+  await Feedback.destroy({ where: {} });
+  await Users.destroy({ where: { email: 'random@gmail.com' } });
+});
+
+describe('Feedback Accomodation', () => {
+  let commentId, tripId;
+  describe('/POST api/v1/accommodations/:id/feedback', () => {
+    it('Should validate Feedback', async () => {
+      const requester = request(server).keepOpen();
+      const [badReq, badReq2] = await Promise.all([
+        requester
+          .post('/api/v1/accommodations/3/feedback')
+          .set('Authorization', `Bearer ${loginToken}`)
+          .send({
+            feedback: ''
+          }),
+        requester
+          .post('/api/v1/accommodations/0/feedback')
+          .set('Authorization', `Bearer ${loginToken}`)
+          .send({
+            feedback: 'feedback'
+          })
+      ]);
+      expect(badReq).to.have.status(400);
+      expect(badReq.body).to.have.property('data');
+      expect(badReq.body.data)
+        .to.have.property('message')
+        .and.to.be.eql('Feedback has to be valid is not allowed to be empty');
+
+      expect(badReq2).to.have.status(404);
+      expect(badReq2.body)
+        .to.have.property('Error')
+        .and.to.be.eql('Accommodation not found');
+    });
+
+    it('Should save feedback', async () => {
+      const res = await request(server)
+        .post('/api/v1/accommodations/3/feedback')
+        .set('Authorization', `Bearer ${loginToken}`)
+        .send(feedback);
+      expect(res).to.have.status(201);
+      expect(res.body).to.have.property('data');
+      expect(res.body.data)
+        .to.have.property('message')
+        .and.to.be.eql('Feedback added successfully');
+    });
+  });
+
+  describe('/GET api/v1/accommodations/:id/feedback', () => {
+    it('Should get feedbacks of accomodation', async () => {
+      const requester = request(server).keepOpen();
+      const [req] = await Promise.all([
+        requester
+          .get('/api/v1/accommodations/3/feedback')
+          .set('Authorization', `Bearer ${loginToken}`)
+      ]);
+      expect(req).to.have.status(200);
+      expect(req.body).to.have.property('data');
+      expect(req.body.data.message).to.be.eql('Feedbacks fetched successfully');
+      expect(req.body.data).to.have.property('data');
```

Comment:
test what is in body.data.data for all test cases.
other cases like:
- Duplicate feedback.
- None existent feedback
- Input validation

as many test cases as possible